zerver/tests/test_slack_importer.py

```diff
@@ -1627,14 +1677,22 @@ def test_channel_message_to_zerver_message_with_threads(
             zerver_message[9][EXPORT_TOPIC_NAME].startswith(expected_thread_5_topic_name)
         )
 
-        ### THREAD 6 CONVERSATION ###
-        # Test various formatting syntaxes in thread topic name
-        expected_thread_6_message_1_content = "**foo** *bar* ~~baz~~ [qux](https://chat.zulip.org)"
-        expected_thread_6_topic_name = (
-            "2019-01-10 **foo** *bar* ~~baz~~ [qux](https://chat.zulip.o…"
-        )
-        self.assertEqual(zerver_message[11]["content"], expected_thread_6_message_1_content)
-        self.assertEqual(zerver_message[11][EXPORT_TOPIC_NAME], expected_thread_6_topic_name)
+    def test_convert_thread_topic_name_with_text_formattings(self) -> None:
+        (
+            zerver_message,
+            _zerver_usermessage,
+            _attachment,
+            _uploads,
+            _reaction,
```

Comment:
This is OK, but a dataclass as the return value here might be a better interface. You can change that as a follow-up if you like the idea.